rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mediasoup"
-version = "0.11.4"
+version = "0.11.5"
```

Comment:
Let's bump this to `0.12.0`, there are a lot of changes accumulated so far and I'm not sure if we made any breaking changes